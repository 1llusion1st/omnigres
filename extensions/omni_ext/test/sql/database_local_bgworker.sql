SELECT omni_ext_test.wait_for_table('local_worker_started');
\d local_worker_started;

CREATE DATABASE another_db_local_worker;
\connect another_db_local_worker
CREATE EXTENSION omni_ext_test CASCADE;
SELECT omni_ext_test.wait_for_table('local_worker_started');
\d local_worker_started;