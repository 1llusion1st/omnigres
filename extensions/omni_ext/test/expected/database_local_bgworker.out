SELECT omni_ext_test.wait_for_table('local_worker_started');
 wait_for_table 
----------------
 t
(1 row)

\d local_worker_started;
      Table "public.local_worker_started"
 Column | Type | Collation | Nullable | Default 
--------+------+-----------+----------+---------

CREATE DATABASE another_db_local_worker;
\connect another_db_local_worker
CREATE EXTENSION omni_ext_test CASCADE;
NOTICE:  installing required extension "omni_ext"
SELECT omni_ext_test.wait_for_table('local_worker_started');
 wait_for_table 
----------------
 t
(1 row)

\d local_worker_started;
      Table "public.local_worker_started"
 Column | Type | Collation | Nullable | Default 
--------+------+-----------+----------+---------

